src/test/java/wooteco/subway/controller/LineAcceptanceTest.java

```diff
@@ -0,0 +1,190 @@
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.transaction.annotation.Transactional;
+import wooteco.subway.AcceptanceTest;
+import wooteco.subway.controller.dto.response.LineResponse;
+
+@DisplayName("지하철 노선 관련 기능")
+@Transactional
+class LineAcceptanceTest extends AcceptanceTest {
+
+    @DisplayName("지하철 노선을 생성한다.")
+    @Test
+    void createLine() {
+        //given
+        Map<String, String> params = new HashMap<>();
+        params.put("color", "bg-red-600");
+        params.put("name", "신분당선");
+
+        //when
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+            .body(params)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when()
+            .post("/lines")
+            .then().log().all()
+            .extract();
+
+        //then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        assertThat(response.header("Location")).isNotBlank();
+        assertThat(response.body().jsonPath().get("name").toString()).isEqualTo("신분당선");
+    }
+
+    @DisplayName("지하철 노선 목록을 조회한다.")
+    @Test
+    void getLines() {
+        // given
+        Map<String, String> param1 = new HashMap<>();
+        param1.put("name", "2호선");
+        param1.put("color", "bg-red-600");
+        ExtractableResponse<Response> createResponse1 = RestAssured.given().log().all()
+            .body(param1)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when()
+            .post("/lines")
+            .then().log().all()
+            .extract();
+
+        Map<String, String> param2 = new HashMap<>();
+        param2.put("name", "3호선");
+        param2.put("color", "bg-red-600");
+        ExtractableResponse<Response> createResponse2 = RestAssured.given().log().all()
+            .body(param2)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when()
+            .post("/lines")
+            .then().log().all()
+            .extract();
+
+        // when
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+            .when()
+            .get("/lines")
+            .then().log().all()
+            .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        List<Long> expectedLineIds = Arrays.asList(createResponse1, createResponse2)
+            .stream()
+            .map(it -> Long.parseLong(it.header("Location").split("/")[2]))
+            .collect(Collectors.toList());
+        List<Long> resultLineIds = response.jsonPath().getList(".", LineResponse.class)
+            .stream()
+            .map(it -> it.getId())
```

Comment:
```suggestion
            .map(LineResponse::getId)
```

가능한 상황에서는 람다 대신 메서드 참조로 나타내는 것이 더 명확합니다 (항상 그런 것은 아니지만요 ㅎㅎ)